reference/apm/observability/apm.md

```diff
@@ -21,19 +21,4 @@ Metrics are another vital source of information when debugging production system
 
 ## Give Elastic APM a try [_give_elastic_apm_a_try]
 
-Use [Fleet-managed APM Server](/reference/fleet/get-started-managed-apm-server.md) to quickly spin up an APM deployment. Want to host everything yourself instead? See [Get started for APM](/solutions/observability/apm/get-started.md).
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+Want to quickly spin up an APM deployment? Refer to [Get started for APM](/solutions/observability/apm/get-started.md). To host everything yourself instead, use [Fleet-managed APM Server](/solutions/observability/apm/apm-server-fleet-managed.md).
```

Comment:
Is there a reason we send folks that want to host everything themselves straight to the Fleet option and not to the top-level Set up page where they can decide if the binary or fleet-managed is right for them? Is it because we don't want to push people to the APM Server binary?